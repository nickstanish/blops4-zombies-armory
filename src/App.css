html, :root {
  font-size: 14pt;
}

html, body {
  padding: 0;
  margin: 0;
  width: 100%;
}

body {
  letter-spacing: normal;
  line-height: 1.3;
  color: #4d4d4d;
  background-color: #efefef;
  font-size: 1rem;
  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";
}

a,
a:visited,
a:active {
  font-weight: 600;
  color: #4d4d4d;
  transition: all 0.12s ease-in-out;
}

a:hover,
a:focus {
  color: #289ccb;
}

.App {
  display: flex;
  flex-direction: column;
  min-height: 100vh;
}

.App-header {
  box-shadow: 2px 2px 6px 4px rgba(0,0,0,0.2);
  padding: 4px 12px;
  font-weight: 600;
  background: rgb(229,127,45);
  /* https://cssgradient.io/ */
  background: linear-gradient(170deg, rgba(229,127,45,1) 0%, rgba(193,106,37,1) 58%, rgba(105,60,24,1) 100%);
  color: #363636;
  padding: 6px 12px;
}

.App-main {
  max-width: 600px;
  width: 90%;
  margin: 0 auto;
  font-size: 16px;
  padding: 12px;
  flex: 1 1 0;
}

.App-footer {
  max-width: 800px;
  margin: 2em auto 1em auto;
  font-size: 12px;
  padding: 0 12px;
}
