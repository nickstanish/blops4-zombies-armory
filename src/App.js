import React from 'react';
import './App.css';
import data from './data.json';
import DataTable from './data-table/DataTable';

function App() {
  return (
    <div className="App">
      <header className="App-header">CoD Black Ops 4 Zombies Armory</header>
      <main className="App-main">
        <DataTable rows={ data.guns } />
      </main>
      <footer className="App-footer">
        Corrections? Open an Issue on&nbsp;
        <a href="https://github.com/nickstanish/blops4-zombies-armory">
          GitHub
        </a>
      </footer>
    </div>
  );
}

export default App;
