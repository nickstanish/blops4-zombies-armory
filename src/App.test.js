import React from 'react';
import { render } from '@testing-library/react';
import App from './App';

jest.mock('./data-table/DataTable', () => () => 'DataTable');

test('renders component', () => {
  const { container } = render(<App />);
  expect(container).toMatchSnapshot();
});
