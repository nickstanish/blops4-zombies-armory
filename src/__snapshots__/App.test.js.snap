// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders component 1`] = `
<div>
  <div
    class="App"
  >
    <header
      class="App-header"
    >
      CoD Black Ops 4 Zombies Armory
    </header>
    <main
      class="App-main"
    >
      DataTable
    </main>
    <footer
      class="App-footer"
    >
      Corrections? Open an Issue onÂ 
      <a
        href="https://github.com/nickstanish/blops4-zombies-armory"
      >
        GitHub
      </a>
    </footer>
  </div>
</div>
`;
